FilteringApp/FilteringApp.Core/Utils/ArrayUtils.cs

```diff
@@ -0,0 +1,73 @@
+﻿using FilteringApp.Core.Models;
+using System.Collections.Generic;
+
+namespace FilteringApp.Core.Utils
+{
+    public static class ArrayUtils
+    {
+        public static List<int> FilterOutResults(UserInput input)
+        {
+            var result = new List<int>();
+            foreach (int v in input.UserArray)
+            {
+                var current = v;
+                while (current > 0)
+                {
+                    if (current % 10 == input.IntValue)
+                    {
+                        result.Add(current);
+                        break;
+                    }
+                    else
+                    {
+                        current /= 10;
+                    }
+                }
+            }
+            return result;
+        }
+        public static int AddArrayElements(int[] array)
+        {
+            var theSum = 0;
+            foreach (int item in array)
+            {
+                theSum += item;
+            }
+            return theSum;
+        }
+        public static ArrayParseResult CheckArray(string[] stringArray)
+        {
+            var array = new int[stringArray.Length];
+            var arrayParseResults = new ArrayParseResult(array);
+            for (int i = 0; i < stringArray.Length; i++)
+            {
+                if (!int.TryParse(stringArray[i], out var value) || string.IsNullOrWhiteSpace(stringArray[i]))
```

Comment:
![image](https://user-images.githubusercontent.com/23036313/142773570-c476c0e3-35f4-4642-8ec3-9e7d79a249ef.png)
You don't actually need second check here, TryParse returns false in case of null or empty or whitespace string

Reply:
Fixed.